src/test/java/com/github/wasiqb/coteafs/selenium/pages/DropdownPage.java

```diff
@@ -0,0 +1,52 @@
+package com.github.wasiqb.coteafs.selenium.pages;
+
+import org.openqa.selenium.By;
+
+import com.github.wasiqb.coteafs.selenium.core.BrowserPage;
+import com.github.wasiqb.coteafs.selenium.core.element.ISelectboxActions;
+import com.github.wasiqb.coteafs.selenium.core.page.ElementKey;
+
+/**
+ * @author Faisal Khatri
+ * @since Jul 19, 2020
+ *
+ */
+public class DropdownPage extends BrowserPage {
+
+	/**
+	 * @author Faisal Khatri
+	 * @since Jul 19, 2020
+	 *
+	 */
+	public enum DropdownKeys implements ElementKey {
+
+		/**
+		 * @author Faisal Khatri
+		 * @since Jul 19, 2020
+		 */
+		OPTION (" ");
```

Comment:
Why the key is empty? You need to specify proper key.

Reply:
Updated.